subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiClientJdkCompatibilityTest.groovy

```diff
@@ -234,7 +237,8 @@ public class ToolingApiCompatibilityBuildAction implements BuildAction<String> {
         }
 
         when:
-        succeeds("buildAction",
+        succeeds("" +
+            "buildAction",
```

Comment:
Is this change intentional? I.e. splitting the string in `"" + "buildAction"`?

Reply:
Thanks for catching! This was not intentional